src/haxelib/api/Installer.hx

```diff
@@ -767,7 +768,8 @@ class Installer {
 			userInterface.log('Installing $library from $url' + (branch != null ? " branch: " + branch : ""));
 			final tag = vcsData.tag;
 			try {
-				vcs.clone(libPath, url, branch, tag, userInterface.log.bind(_, Debug));
+				FsUtils.deleteRec(libPath);
```

Comment:
Is `FsUtils.deleteRec` necessary here? It looks like this is currently handled before any calls to `doVcsClone`?